templates/index.html

```diff
@@ -23,7 +23,16 @@ <h2>Items</h2>
     <label for="title">title:</label>
       <input type="text" id="title" name="title">
     </li>
-    <button type="submit">Send your title</button>
+    <button type="submit">Add Card</button>
   
    </form>
+
+   <form action="/move_Item" method="post">
+    
+    <label for="title">CardName:</label>
+      <input type="text" id="cardname" name="cardname">
+      <button type="submit">Move Card</button>
+    
+   </form>
```

Comment:
This works but isn't very good UX right? As I said before you could have a button next to each item in the list if you put something like this in the for loop on line 16. (And if you also update app.py to use `request.form['card_id']`)

```

<form action="/move_Item" method="post">
  <input type="hidden" id="card_id" value="{{card.id}}">
  <button type="submit">Reviewed</button>
</form>
```